### PR TITLE
Testutils pass, on to develop

### PR DESCRIPTION
**Pull request by acieroid**

### CODE REVIEW

File: src/app/games/kamisado/tests/kamisado.component.spec.ts [hunk 1]
```diff
@@ -60,9 +60,12 @@ describe('KamisadoComponent', () => {
         ];
         const state: KamisadoState =
             new KamisadoState(6, KamisadoColor.RED, MGPOptional.of(new Coord(0, 7)), false, board);
+
+        // When displayign the board
```
Comment (MartinREMY42) [inline src/app/games/kamisado/tests/kamisado.component.spec.ts]: displayigneh

File: src/app/games/kamisado/tests/kamisado.component.spec.ts [hunk 2]
```diff
@@ -48,6 +47,7 @@ describe('KamisadoComponent', () => {
         expect(componentTestUtils.getComponent().chosen.isAbsent()).toBeTrue();
     }));
     it('should allow to pass if stuck position', fakeAsync(async() => {
+        // Given a board with a stuck piece being the one that has to play
```
Comment (MartinREMY42) [inline src/app/games/kamisado/tests/kamisado.component.spec.ts]: that has to move or thas has to be played

File: src/app/games/kamisado/tests/kamisado.component.spec.ts (hunk 1 above)
Comment (acieroid) [reply src/app/games/kamisado/tests/kamisado.component.spec.ts]: done

File: src/app/games/kamisado/tests/kamisado.component.spec.ts (hunk 2 above)
Comment (acieroid) [reply src/app/games/kamisado/tests/kamisado.component.spec.ts]: gedone

**Pull request merged**